src/devtools/client/debugger/src/components/QuickOpenModal.js

```diff
@@ -33,12 +31,14 @@ import {
   parseLineColumn,
   formatShortcutResults,
   formatSources,
-  formatProjectSymbols,
 } from "../utils/quick-open";
 import Modal from "./shared/Modal";
 import SearchInput from "./shared/SearchInput";
 import ResultList from "./shared/ResultList";
 import { trackEvent } from "ui/utils/telemetry";
+import { ThreadFront } from "protocol/thread";
+import { getSourceIDsToSearch } from "devtools/client/debugger/src/components/FullTextSearch/search";
+import { getGlobalFunctions, isGlobalFunctionsLoading } from "../reducers/ast";
```

Comment:
do we need this?

import { ThreadFront } from "protocol/thread";
import { getSourceIDsToSearch } from "devtools/client/debugger/src/components/FullTextSearch/search";

Reply:
Just deleted it 👍